tests/foreman/cli/test_role.py

```diff
@@ -202,6 +205,59 @@ def test_negative_list_filters_without_parameters(self):
         self.assertRegex(
             err.exception.msg, 'At least one of options .* is required')
 
+    @tier1
+    def test_positive_list_filters_with_pagination(self):
+        """Make sure filters list can be displayed with different items per
+        page value
+
+        :id: b9c7c6c1-70c2-4d7f-8d36-fa8613acc865
+
+        :BZ: 1428516
+
+        :expectedresults: `per-page` correctly sets amount of items displayed
+            per page, different `per-page` values divide a list into correct
+            number of pages
+
+        :CaseImportance: Critical
+        """
+        role = make_role()
+        res_types = iter(PERMISSIONS.keys())
+        permissions = []
+        # Collect more than 20 different permissions
+        while len(permissions) <= 20:
```

Comment:
That's a good point but no worries, i used `next(res_types)` in a loop - in case satellite doesn't have enough permissions `StopIteration` will be risen.